.github/workflows/ci-e2e-checks.yaml

```diff
@@ -9,25 +9,25 @@ on:
       - master
 
 jobs:
- 
+
   e2e-tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
We can try another version, but using the latest at least now we have been having problems with `yq` package. That's why I reverted to the Ubuntu 18.04 version.

`Error: unknown command "read" for "yq"
[6](https://github.com/mariomcosta/imageswap-webhook/runs/6396696629?check_suite_focus=true#step:11:6)
Run 'yq --help' for usage.`

Reply:
This is because the newer Ubuntu image installs yq v4.x and the syntax has changed slightly. They made "read" the default function without any arguments passed. In most cases you just need to change `yq read` to just `yq`